packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -43,6 +49,17 @@ class _ChildEntry {
   }
 }
 
+/// Signature for builders used to generate custom layouts for
+/// [PageTransitionSwitcher].
+///
+/// The builder should return a widget which contains the given children, laid
+/// out as desired. It must not return null. The builder should be able to
+/// handle an empty list of `_activeEntries`.
```

Comment:
We should define "active entries" here, since users might not get what these refer to. The only reference they have without peeking into the implementation is the concept of an "old" and "new" child widget, so this concept will need to be clarified.